src/connection-monitor-preview/azext_connection_monitor_preview/_params.py

```diff
@@ -0,0 +1,304 @@
+                   help='Value indicating whether path evaluation with trace route should be disabled. '
+                        'false is default.',
+                   arg_type=get_three_state_flag())
+        # ICMP protocol configuration
+        c.argument('test_config_icmp_disable_trace_route',
+                   options_list='--icmp-disable-trace-route',
+                   help='Value indicating whether path evaluation with trace route should be disabled. '
+                        'false is default.',
+                   arg_type=get_three_state_flag())
+        # HTTP protocol configuration
+        c.argument('test_config_http_port',
+                   options_list='--http-port',
+                   help='The port to connect to',
+                   type=int)
+        c.argument('test_config_http_method',
+                   options_list='--http-method',
+                   help='The HTTP method to use',
+                   arg_type=get_enum_type(HTTPConfigurationMethod))
+        c.argument('test_config_http_path',
+                   options_list='--http-path',
+                   help='The path component of the URI. For instance, "/dir1/dir2"')
+        c.argument('test_config_http_valid_status_codes',
+                   options_list='--http-valid-status-codes',
+                   help='Space-separated list of HTTP status codes to consider successful. '
+                        'For instance, "2xx 301-304 418"',
+                   nargs='+')
+        c.argument('test_config_http_prefer_https',
+                   options_list='--https-prefer',
+                   help='Value indicating whether HTTPS is preferred '
+                        'over HTTP in cases where the choice is not explicit',
+                   arg_type=get_three_state_flag())
+
+    # Argument Group for test group to create a V2 connection monitor
+    with self.argument_context('network watcher connection-monitor',
+                               arg_group='V2 Test Group',
+                               min_api='2019-11-01') as c:
+        c.argument('test_group_name',
+                   help='The name of the connection monitor test group',
+                   default='DefaultTestGroup')
+        c.argument('test_group_disable',
+                   help='Value indicating whether test group is disabled. false is default.',
+                   arg_type=get_three_state_flag())
+
+    # Argument Group for output to create a V2 connection monitor
+    with self.argument_context('network watcher connection-monitor',
+                               arg_group='V2 Output',
+                               min_api='2019-11-01') as c:
+        c.argument('output_type',
+                   help='Connection monitor output destination type. Currently, only "Workspace" is supported',
+                   arg_type=get_enum_type(OutputType))
+        c.argument('workspace_ids',
+                   help='Space-separated list of ids of log analytics workspace',
```

Comment:
could it be `list of resource ids of`?

Reply:
is it possible to support both name/id?

Reply:
As we discussd last night, it's not worth to support name to specify log analytics workspace.